package.json

```diff
@@ -3,9 +3,9 @@
   "version": "2.0.0",
   "private": true,
   "scripts": {
-    "ganache": "ganache-cli -i 10",
+    "ganache": "ganache-cli -i 10 -m 'describe uncle will various ankle film brother pelican apple congress animal segment'",
     "postinstall": "typechain --target=web3-v1 contracts/build/contracts/XDVToken.json --outDir=server/types/typechain",
-    "demo": "npm run truffle exec scripts/demo.js --workspace=contracts"
+    "demo": "truffle exec contracts/scripts/demo.js"
```

Comment:
Please restore the old versions. The documentation is already written with them in mind.